recipes/cairo/all/conanfile.py

```diff
@@ -222,11 +224,9 @@ def package(self):
         tools.remove_files_by_mask(self.package_folder, "*.la")
 
         self.copy("COPYING*", src=self._source_subfolder, dst="licenses", keep_path=False)
-        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
-
+        files.rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
 
     def package_info(self):
-        self.cpp_info.components["cairo_"].names["pkg_config"] = "cairo"
```

Comment:
Please keep all these `pkg_config` names (you can translate them to `set_property("pkg_config_name", foo)`), so we can track these names are properly handled

To fix your issue, add a `pkg_config_name` at self.cpp_info level like `cairo-all-do-not-use`